hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/io/ECKeyOutputStream.java

```diff
@@ -377,7 +391,7 @@ private int writeToOutputStream(BlockOutputStreamEntry current, long len,
       // the buffers
       Preconditions.checkState(len <= config.getStreamBufferMaxSize());
       int dataWritten = (int) (current.getWrittenDataLength() - currentPos);
-      writeLen = dataWritten;
+      //writeLen = dataWritten;
```

Comment:
Probably should remove the commented line.

Reply:
done.